client/src/components/PropertyInspectorComponent/PropertyInspectorComponent.test.js

```diff
@@ -53,7 +54,10 @@ test("render component", async () => {
     render(<PropertyInspectorComponent isOpen />, container);
   });
 
-  eventService.publishSelection(twinSelection);
+  act(()=> {
```

Comment:
Add space between `()` and `=>`. We should see why the linter isn't picking up stuff like this for us!

Reply:
Added the space, we might need to adjust the lint config for test files